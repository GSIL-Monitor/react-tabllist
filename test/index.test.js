import pkg from '../package.json'

let reactTabllist = require('../src')

describe('react-tabllist dist files', () => {
  it('exports modules correctly', () => {
    expect(Object.keys(reactTabllist)).toMatchSnapshot()
  })
})
