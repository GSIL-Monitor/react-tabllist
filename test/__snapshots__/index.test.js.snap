// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-tabllist dist files exports modules correctly 1`] = `
Array [
  "default",
]
`;
